docs/detections/value-list-exceptions.asciidoc

```diff
@@ -37,27 +37,54 @@ act as delimiters.
 =========================
 
 . Go to *Rules* -> *Detection rules (SIEM)*.
-. Click *Import value lists*. The *Import value lists* window opens.
+. Click *Manage value lists*. The *Manage value lists* window opens.
 +
 [role="screenshot"]
-image::images/upload-lists-ui.png[Import value lists flyout,75%]
+image::images/upload-lists-ui.png[Manage value lists flyout,75%]
 
 . Select the list type (*Keywords*, *IP addresses*, *IP ranges*, or *Text*) from the *Type of value list* drop-down.
 . Drag or select the `csv` or `txt` file that contains the values.
 . Click *Import value list*.
 
-NOTE: When the name of the file you are importing already exists, the values in
-the new file are appended to the previously imported values.
+NOTE: If you import a file with a name that already exists, a new list is not created. The imported values are added to the existing list instead.
 
-[[edit-value-lists]]
+[[manage-value-lists]]
 [discrete]
 === Manage value lists
 
-To view, delete, or export existing value lists:
+You can edit, remove, or export existing value lists. 
+
+[[edit-value-lists]]
+[discrete]
+==== Edit value lists
+
+. Go to **Rules** → **Detection rules (SIEM)**.
+. Click **Manage value lists**. The **Manage value lists** window opens. 
+. In the **Value lists** table, click the value list you want to edit.
+. Do any of the following:
+
+** **Filter items in the list**: Use the KQL search bar to find values in the list. Depending on your list's type, you can filter by the `keyword`, `ip_range`, `ip`, or `text` fields. For example, to filter by Gmail addresses in a value list of the `keyword` type, enter `keyword:*gmail.com` into the search bar. 
++
+You can also filter by the `updated_by` (for example, `updated_by:testuser`), or `updated at` (for example, `updated_at < now`) fields.  
```

Comment:
```suggestion
You can also filter by the `updated_by` field (for example, `updated_by:testuser`), or the `updated at` field (for example, `updated_at < now`).  
```
Had to re-read the sentence a couple times to get it. I think this scans more easily